src/EnvRegistry.ts

```diff
@@ -0,0 +1,49 @@
+import { from } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { messages } from '@capsulajs/capsulajs-configuration-service/lib/utils';
```

Comment:
don't you think that maybe it's better to export all the modules that are required externally in the index file of the package and set the entry point in lib directory? so that you can import it as 
import { messages } from '@capsulajs/capsulajs-configuration-service';

Reply:
I will create new PR for it in conf service